README.md

```diff
@@ -6,37 +6,108 @@ Membrane Curvature
 [![codecov](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main/graph/badge.svg)](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main)
 [![docs](https://readthedocs.org/projects/membrane-curvature/badge/?version=latest)](https://membrane-curvature.readthedocs.io/en/latest/)
 
+<img src="docs/source/_static/PM_Membrane_EBO.png" alt="solution_posres"
+width="650"/>
 
-This is an [MDAnalysis][mdanalysis] module to calculate membrane curvature from 
-molecular dynamics simulations. 
+MembraneCurvature is an [MDAnalysis] tool to calculate membrane curvature from 
+Molecular Dynamics simulations. 
 
-The MDAkit for membrane curvature analysis is part of the [Google Summer of Code][GSoC] program 
-and it is linked to a [Code of Conduct][code_of_conduct].
+Features
+--------------
+
+With MembraneCurvature you can:
+
+- Calculate mean and Gaussian curvature from MD simulations.
+- Derive 2D curvature profiles.
+- Live a happier life.
+
+
+Installation
+--------------
+
+To install from source:
```

Comment:
So once you do your first release I assume the aim is that this will become available via pip / conda (pinging @lilyminium @orbeckst @fiona-naughton ) - I guess these instructions can be updated after the fact?